app/views/content/is-teaching-right-for-me/all-subjects.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Look at the subjects you could teach
+description: |-
+  Look at the subjects you could teach and choose what teacher training course you would like to do.
+
+keywords:
+  - teacher training
+  - teaching maths
+  - teaching history
+  - teaching science
+  - teaching art
+  - art and design
+
+
+---
+
+Explore the A-Z of all the subjects you could choose for your teacher training course.
+
+
+## Ancient Greek
+
+As an ancient Greek teacher you'll teach pupils about what life was like in ancient Greece, and the achievements of the people and their impact on the western world. 
+
+<a href="https://www.find-postgraduate-teacher-training.service.gov.uk/results?age_group=secondary&applications_open=true&can_sponsor_visa=false&has_vacancies=true&l=2&subjects%5B%5D=A1&visa_status=false">Search for ancient Greek teacher training courses</a>.
```

Comment:
@caitlin-reidy would it be better to say Find rather than Search for? Just thinking about Find and Apply and if we're using the same language?

Reply:
@caitlin-reidy but I'm not sure if at this point it's the right place to have a link to find courses? At this stage we don't know if there's a user need/ if they're thinking about courses when we're telling them about subject options? When Amilcar and Rose saw the page they both thought it was an A-Z list of courses rather than subject pages. So potentially it could be confusing for users too?